### PR TITLE
Player name change 

### PR DESCRIPTION
**Pull request by filiptrplan**
* ability to change the player name with key `N`
* tested it myself, looks good

### CODE REVIEW

File: world-client/src/features/me.js [hunk 1]
```diff
@@ -152,7 +163,11 @@ export default class Me extends Feature {
       wop.me.id = message.id;
       wop.me.setName(message.name);
     }
+    if (message.type == 'player.changename' && message.id == wop.me.id) {
+      wop.me.setName(message.name);
+    }
 
+// haha Adam je glup
```
Comment (davidmohar) [inline world-client/src/features/me.js]: Ha ha, very funny, please remove

File: world-server/src/world/player/PlayerHandlers.ts [hunk 2]
```diff
@@ -26,6 +26,9 @@ export class PlayerHandler implements Messages.MsgHandler {
             Logger.warn(`Received player message does not extend PlayerMessage`);
             return;
         }
+        console.log("some message" + message.type);
```
Comment (davidmohar) [inline world-server/src/world/player/PlayerHandlers.ts]: Remove calls to `console.log` used for debugging

**Review by davidmohar — CHANGES_REQUESTED** [review]

**Review by jakakonda — DISMISSED** [review]
Glejte komentarje od Davida.

File: world-client/src/features/me.js (hunk 1 above)
Comment (filiptrplan) [reply world-client/src/features/me.js]: Fixed it for them :)

File: world-server/src/world/player/PlayerHandlers.ts (hunk 2 above)
Comment (filiptrplan) [reply world-server/src/world/player/PlayerHandlers.ts]: Fixed it for them :)

**Review by davidmohar — APPROVED** [review]

**Pull request merged**